modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -45,7 +44,11 @@ def create
       ).call
 
       unless status == :ok
-        log_exception_to_sentry(EvidenceSubmissionRequestValidatorError.new(error), {}, {}, :warn)
+        req_validator_error = EvidenceSubmissionRequestValidatorError.new(error)
+        error_details = { error_message: req_validator_error.message }
+        error_details[:back_trace] = req_validator_error.backtrace.join('\n') unless req_validator_error.backtrace.nil?
+        Rails.logger.warn('Nod Evidence Submission Validation Error', error_details)
```

Comment:
When I looked into this a bit more, I discovered that the `backtrace` of the `EvidenceSubmissionRequestValidatorError` is always `nil` (here, and in the other evidence submission controllers as well). Therefore, L49 isn't really doing anything and can be removed.